src/_posts/whats-new-in-0.16.0.md

```diff
@@ -0,0 +1,128 @@
+
+## Performance
+
+### Allow the initial test suite to be skipped (`--skip-initial-test-suite`)
+
+Infection's first step is executing your project's test suite. If it fails, further mutation process is useless, because any Mutant can be marked as Killed just because of the failing tests for the original code.
+
+That's why passing test suite is essential.
+
+If you have already run the test suite to ensure it is passing, and you are providing coverage using `--coverage` then you can use `--skip-initial-test-suite` option to increase performance by not running the entire test suite again.
+
+<p class="tip">Note that it is extremely important that you ensure the test suite is passing when using this option, otherwise test failures would appear like caught mutations and those mutations may be reported incorrectly.</p>
+
+### Other performance improvements
+
+TBD
+
+* https://github.com/infection/infection/pull/1177
+* https://github.com/infection/infection/pull/1172
+* https://github.com/infection/infection/pull/1106
+* https://github.com/infection/infection/pull/1082
+
+## Splitting Infection by separate Packages
+
+Infection has been split by different packages. For example, we have extracted `PhpSpec` and `Codeception` adapter out of the core.
+
+You can read an very detailed explanation about why was it needed in [this RFC](https://github.com/infection/infection/issues/922).
+
+In a nutshell, this is useful both for the end user and for maintainers:
+
+* you will not download not needed code if you don't use those test frameworks
+* you will not need to upgrade Infection if for example bug is fixed in the test framework you don't use
+* now Test Frameworks have clear dependencies and requirements
+
+Don't worry, if you are using Codeception or PhpSpec, these packages will be automatically installed at the first Infection execution after upgrading to 0.16.0.
+
+List of new packages:
+
+* [`infection/codeception-adapter`](https://github.com/infection/codeception-adapter)
+* [`infection/phpspec-adapter`](https://github.com/infection/phpspec-adapter)
+* [`infection/abstract-testframework-adapter`](https://github.com/infection/abstract-testframework-adapter) - interfaces for any testframework adapter
+* [`infection/extension-installer`](https://github.com/infection/extension-installer) - autodiscovery of installed adapters. No need to add them to `infection.json`, just install a Composer package and you are done. Inspired by PHPStan and Psalm plugin system.
+* [`infection/include-interceptor`](https://github.com/infection/include-interceptor) - Stream Wrapper. Allows to replace included (autoloaded) file with another one (magic that replaces your code with a mutated one) 
```

Comment:
What do you think about keeping this list sorted by package names?

Reply:
thought about it, but they are groupd "by meaning" right now, which IMO better here